frontend/index.js

```diff
@@ -1,34 +1,23 @@
 "use strict"
 
-var wasm
+let wasmModule, wasmInst
+let sourcePtr, sourceLength
+const runButton = document.getElementById("run")
 
 // initWasm loads bytecode and initialises execution environment.
 function initWasm() {
-  const go = new Go() // see wasm_exec.js
-  const evyEnv = {
-    jsPrint,
-    move,
-    line,
-    width,
-    circle,
-    rect,
-    color,
-  }
-  go.importObject.env = Object.assign(go.importObject.env, evyEnv)
-  WebAssembly.instantiateStreaming(fetch("evy.wasm"), go.importObject)
+  WebAssembly.compileStreaming(fetch("evy.wasm"))
     .then(function (obj) {
-      wasm = obj.instance
-      go.run(wasm)
+      wasmModule = obj
     })
     .catch((err) => {
       console.error(err)
     })
```

Comment:
nit: This could be formatted with fewer syntactical elements I think:
```
    .then(obj => wasmModule = obj)
    .catch(err => console.error(err))
```

Reply:
done.